src/main/webapp/app/admin/metrics/blocks/jvm-memory/jvm-memory.component.html

```diff
@@ -1,5 +1,5 @@
 <h4 jhiTranslate="metrics.jvm.memory.title"></h4>
-@if (!updating && jvmMemoryMetrics) {
+@if (!updating && jvmMemoryMetrics && Object.keys(this.jvmMemoryMetrics!).length > 0) {
```

Comment:
is this really needed? looks quite ugly!

Reply:
can we not simply use?

`@if (!updating && jvmMemoryMetrics && jvmMemoryMetrics.length > 0) {`

Reply:
I aggree that it is pretty ugly. Unfortunately using `.length` that doesn't work here, as jvmMemoryMetrics isn't an array, but an object (basically a string map). So that's not possible here. 
But: it turns out that we do not need this here any more. `jvmMemoryMetrics` is always truthy here (except if it really **is** undefined) even when it contains no jvm data, and then the `@for` loops over an empty map, which then just displays nothing.

Previously I had added an `@else` to display a message that no data was received, but I did remove this, so the awkward length check is not needed any more. If we wanted to show such a warning to the user, instead of showing nothing, we would need some sort of length check there.